### PR TITLE
update Readme.md file

### PR DESCRIPTION
**Pull request by jaswindersingh2601**
I just add h2 heading in Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# github-practices
\ No newline at end of file
+# github-practices
+## Hello Jaswinder There, I commit these changes to github-practices.
```
Comment (kabir0x23) [inline README.md:2]:
```suggestion
## Hello there, I commit these changes to github-practices.
```

File: README.md [hunk 2]
```diff
@@ -1 +1,4 @@
...
+
+
```
Comment (kabir0x23) [inline README.md:4]: remove the extra line, keep only one line.

**Review by kabir0x23 — CHANGES_REQUESTED** [review]
Please give a look at your code

File: README.md (hunk 2 above)
Comment (jaswindersingh2601) [reply README.md]: last lines got removed.

**Review by kabir0x23 — APPROVED** [review]
Looks good

**Pull request merged**